### DISCUSSION TITLE
Should we display the public IP address of Transmission in UI

### DISCUSSION THREAD

**Discussion by tearfur** [transmission/transmission#5618] [Ideas]
Now that we have the global IP cache implemented (#5329, #5498, #5510), is it a good idea to show in the UI (e.g. in a pop-up opened by a menu item) the public IP that Transmission is being seen as?

I imagine this would be useful for users to troubleshoot connectivity problems, or to ensure that they are binding to the correct interface etc.

Just throwing this out there for discussion.

**Comment by Pentaphon** [comment]
Sure, probably next to the port setting in preferences  if anybody wanted to open a PR for that.

**Comment by ckerr** [comment]
Yeah, if we were to do this it would probably make sense near the "port open" / "port closed" status.

**Comment by dantearmok** [comment]
Yes! Please implement this!

**Comment by DuvelCorp** [comment]
very good idea since we mostly all use VPN with killswitch :)

**Comment by tearfur** [comment]
Converted to an issue at #5785 for now since it seems like there are interest to this feature. 

It's not a high priority for me, so I've marked the issue as `pr welcome` to see if somebody else wants to pick it up.
